qurator/eynollah/ocrd-tool.json

```
@@ -0,0 +1,54 @@
+{
+  "version": "0.0.1",
+  "git_url": "https://github.com/qurator-spk/eynollah",
+  "tools": {
+    "ocrd-eynollah-segment": {
+      "executable": "ocrd-eynollah-segment",
+      "categories": ["Layout analysis"],
+      "description": "Segment page into regions and lines and do reading order detection with eynollah",
+      "input_file_grp": ["OCR-D-IMG", "OCR-D-SEG-PAGE", "OCR-D-GT-SEG-PAGE"],
+      "output_file_grp": ["OCR-D-SEG-LINE"],
+      "steps": ["layout/segmentation/region", "layout/segmentation/line"],
+      "parameters": {
+        "models": {
+          "type": "string",
+          "format": "file",
+          "cacheable": true,
+          "description": "Path to directory containing models to be used (See https://qurator-data.de/eynollah)",
+          "required": true
+        },
+        "dpi": {
+          "type": "number",
+          "format": "float",
+          "description": "pixel density in dots per inch (overrides any meta-data in the images); disabled when <= 0",
+          "default": -1
+        },
+        "full_layout": {
+          "type": "boolean",
+          "default": true,
+          "description": "Try to detect all elements, including drop-caps and marginalia"
+        },
+        "curved_line": {
+          "type": "boolean",
+          "default": false,
+          "description": "if this parameter set to true, this tool will try to return contoure of textlines instead of rectabgle bounding box of textline. This should be taken into account that with this option the tool need more time to do process."
+        },
+        "allow_enhancement": {
+          "type": "boolean",
+          "default": true,
+          "description": "if this parameter set to true, this tool would check that input image need resizing and enhancement or not. If so output of resized and enhanced image and corresponding layout data will be written in out directory"
```

```suggestion
          "description": "if the input image has less than 300 DPI, then upscale and enhance"
```

Note: I have _not_ seen any AlternativeImage output in the OCR-D wrapper for this. (Probably not possible ATM due to missing `@scale` representation in PAGE anyway.)

---

Yes, it's purely internal at the moment, not serialized.